src/types/line.js

```diff
@@ -113,10 +127,12 @@ function drawLabel(ctx, line) {
 
 	const {width, height} = measureLabel(ctx, label);
 	const pos = calculateLabelPosition(line, width, height);
+	const rotation = line._obliqueLine && !label.rotation && !!label.autoRotation ? calculateAutoRotation(line, chartArea) : label.rotation * PI / 180;
```

Comment:
So this could be `const rotation = label.rotation == 'auto' ? calculateAutoRotation(line, chartArea) : toRadians(label.rotation);`